Casks/b/bing-wallpaper.rb

```diff
@@ -1,5 +1,5 @@
 cask "bing-wallpaper" do
-  version "1.1.3,24"
+  version "1.1.3"
   sha256 "0c7194da0c12c92c8a4edb8e298ed4e523b3d2db5610cdc43a9255e92f58ebfa"
 
   url "https://download.microsoft.com/download/9/4/a/94aa841f-5578-4856-8a8c-83ff78eb2d1f/Installer/#{version.csv.first}/MSN/Flight2/MW021/Bing%20Wallpaper.pkg"
```

Comment:
```suggestion
  url "https://download.microsoft.com/download/9/4/a/94aa841f-5578-4856-8a8c-83ff78eb2d1f/Installer/#{version}/MSN/Flight2/MW021/Bing%20Wallpaper.pkg"
```

I'm not sure if there's an official position on `version.csv.first` vs. `version` for `version` values that aren't comma-separated but we can use `#{version}` if this `version` format remains the same going forward.